### PR TITLE
Bug 1338324 - Base support for Windows 7 in the datacenter

### PR DESCRIPTION
**Pull request by markcor**

### CODE REVIEW

File: userdata/DynamicConfig.ps1 [hunk 1]
```diff
@@ -6,46 +6,53 @@ file, You can obtain one at http://mozilla.org/MPL/2.0/.
...
-    SetScript = {
-      if ("${env:ProgramFiles(x86)}") {
-        $gpg = ('{0}\GNU\GnuPG\pub\gpg.exe' -f ${env:ProgramFiles(x86)})
-      } else{
-        $gpg = ('{0}\GNU\GnuPG\pub\gpg.exe' -f $env:ProgramFiles)
+  
+  # SourceRepo is in place to toggle between production and testing environments
+	$SourceRepo = mozilla-releng
+	
+  If ($LocationType -eq "AWS") {
```
Comment (grenade) [inline userdata/DynamicConfig.ps1]: $LocationType will need to be set here in DynamicConfig as well. Something similar to how it's handled in rundsc.ps1 is fine (if (Get-Service "Ec2Config" -ErrorAction SilentlyContinue){...})

File: userdata/rundsc.ps1 [hunk 2]
```diff
@@ -367,6 +377,23 @@ function New-LocalCache {
   }
 }
 
+# SourceRepo is in place to toggle between production and testing environments
+$evn:SourceRepo = mozilla-releng
```
Comment (grenade) [inline userdata/rundsc.ps1]: was this meant as $SourceRepo or maybe $env:SourceRepo ? I don't think it's doing anything as is.

File: userdata/rundsc.ps1 [hunk 3]
```diff
@@ -627,10 +661,12 @@ if ($rebootReasons.length) {
           $priorityClass = $gwProcess.PriorityClass
           $gwProcess.PriorityClass = [Diagnostics.ProcessPriorityClass]::AboveNormal
           Write-Log -message ('process priority for generic worker altered from {0} to {1}.' -f $priorityClass, $gwProcess.PriorityClass) -severity 'INFO'
+          Stop-Service $UpdateService
         }
       }
     }
   }
 }
+Write-Log -message "Disabling Windows Update service"
```
Comment (grenade) [inline userdata/rundsc.ps1]: this seems out of place here. can we get it inside the code block that does the disable?

**Comment by markcor** [comment]
I made the changes regarding the variables. 

I removed the log message since the service is being disabled in the manifest.

**Comment by markcor** [comment]
Grenade could you merge if it all looks good?

**Pull request merged**